examples/stable-diffusion/training/train_text_to_image_lora.py

```diff
@@ -0,0 +1,1310 @@
+#!/usr/bin/env python
+# coding=utf-8
```

Comment:
can you add the lora training to train_text_to_image.py with argument?

Reply:
Yes, we can join 2 scripts into One

Reply:
Done